diff --git a/org/mozilla/jss/pkcs11/PKCS11Constants.java b/org/mozilla/jss/pkcs11/PKCS11Constants.java
index 7e3f3c0..81e0e02 100644
--- a/org/mozilla/jss/pkcs11/PKCS11Constants.java
+++ b/org/mozilla/jss/pkcs11/PKCS11Constants.java
@@ -5632,7 +5632,7 @@ public interface PKCS11Constants {
      *
      * Source file: /usr/include/nss3/pkcs11t.h
      */
-    public static final long CKR_PUBLIC_KEY_INVALID = 0x0000001BL;
+    public static final long CKR_PUBLIC_KEY_INVALID = 0x000001B9L;
 
     /**
      * Content automatically generated; see NSS documentation for more information.

